### PR TITLE
remove low leaf area test

### PR DESCRIPTION
**Pull request by peter-devoil**
Working on #1950

Test to find damage

### CODE REVIEW

**Comment by BrianCollinss** [comment]
Don't you think it's better to move the test or the error handler to after **lai += dltLAI;**?

**Comment by jbrider** [comment]
@BrianCollinss The piece of code that was replaced wasn't checking if dltLAI was larger than LAI. We're not sure what it's purpose was - it's been there a long time. The exception would show if the code was there to keep LAI > 0 - but it doesn't look like that was it's intent.

**Comment by BrianCollinss** [comment]
You're right. I am just thinking that if **if(lai < 0.001) lai = 0.0** is left in place, **lai += dltLAI** could make the LAI<0, because **lai=0**!

File: Model/Sorghum/Leaf.cpp [hunk 1]
```diff
@@ -351,42 +352,91 @@ double Leaf::calcLaiSenescenceFrost(void)
...
          sprintf(msg,"Frost Event: (Non Fatal) \n");
          scienceAPI.write(msg);
-         sprintf(msg, "\t\tMin Temp      = %.2f \t\t Senesced LAI    = %.2f\n", plant->today.minT, lai - 0.01);
+         sprintf(msg, "\t\tMin Temp      = %.2f \t\t Senesced LAI    = %.2f\n", 
+                 plant->today.minT, lai - 0.01);
          scienceAPI.write(msg);
-
-         senescedLAI =  Max(0.0,lai - 0.1);
+         // FIXME - which is it? 0.01 or 0.1? If it's a fraction, then shouldn't we 
+         //         multiply by 0.9?
```
Comment (peter-devoil) [inline Model/Sorghum/Leaf.cpp:359]: @jbrider - query here. Fractional loss, or 0.1?

Comment (jbrider) [reply Model/Sorghum/Leaf.cpp:359]: It was 0.1 to remove most of the leaf but retain enough that it wasn't killed due to loss of leaf.

**Comment by peter-devoil** [comment]
Abandoned, another go at https://github.com/APSIMInitiative/APSIMClassic/pull/1979

**Pull request closed without merging**